app/controllers/dishes_controller.rb

```diff
@@ -0,0 +1,6 @@
+class DishesController < ApplicationController
+  def show
+    @dish = Dish.find(params[:id])
+    @all_ingredient_names = @dish.all_ingredient_names
```

Comment:
Since you made a method for this, you don't need to create another ivar here. Instead, I would just make the `@dish` ivar and then from the view, you would then call `@dish.all_ingredient_names`